### ISSUE TITLE
DefaultSchemaRaw has no field minimum/maximum

### ISSUE THREAD

**Issue by RaulTrombin** [paperclip-rs/paperclip#532]
Hello,
is there some special feature or decorator to enable new feature related to min/max?

I was using:
paperclip = { version = "0.8.2" , features = ["actix4", "swagger-ui", "uuid"] }

Then updating to sync with latest version it raises the error:
```
error[E0560]: struct `DefaultSchemaRaw` has no field named `maximum`
   --> /home/raulvt/.cargo/registry/src/index.crates.io-6f17d22bba15001f/paperclip-core-0.7.0/src/v2/schema.rs:424:13
    |
424 |             maximum: T::max(),
    |             ^^^^^^^ `DefaultSchemaRaw` does not have this field
    |
    = note: available fields are: `name`, `reference`, `title`, `description`, `example` ... and 5 others

error[E0560]: struct `DefaultSchemaRaw` has no field named `minimum`
   --> /home/raulvt/.cargo/registry/src/index.crates.io-6f17d22bba15001f/paperclip-core-0.7.0/src/v2/schema.rs:425:13
    |
425 |             minimum: T::min(),
    |             ^^^^^^^ `DefaultSchemaRaw` does not have this field
    |
    = note: available fields are: `name`, `reference`, `title`, `description`, `example` ... and 5 others
```

```
impl<T: TypedData> Apiv2Schema for T {
    fn raw_schema() -> DefaultSchemaRaw {
        DefaultSchemaRaw {
            data_type: Some(T::data_type()),
            format: T::format(),
            max: T::max(),
            minimum: T::min(),
            ..Default::default()
        }
    }
}
```

removing the
```
            max: T::max(),
            minimum: T::min(),
```
solve the compilation issue, I don't think I'm using the new feature, though.

**Comment by tiagolobocastro** [comment]
I had forgotten to update paperclip-macros. Could you try updating paperclip-macros to 0.6.4?

**Comment by tiagolobocastro** [comment]
Also published v0.9.1
Could you please confirm this resolves things for you?

**Comment by RaulTrombin** [comment]
@tiagolobocastro , the current master worked locally, i'm going to try the newer version now

**Comment by RaulTrombin** [comment]
> Also published v0.9.1 Could you please confirm this resolves things for you?

Solved the issue.

**Comment by tiagolobocastro** [comment]
> > Also published v0.9.1 Could you please confirm this resolves things for you?
> 
> Solved the issue.

Awesome, thank you.
Perhaps we need to add some kind of publish test to prevent this from happening again (me forgetting to update the crate version on the toml)

**Comment by RaulTrombin** [comment]
> > > Also published v0.9.1 Could you please confirm this resolves things for you?
> > 
> > 
> > Solved the issue.
> 
> Awesome, thank you. Perhaps we need to add some kind of publish test to prevent this from happening again (me forgetting to update the crate version on the toml)

Maybe we can export some api_specs and validate then using schemathesis?
https://github.com/schemathesis/schemathesis
